### PR TITLE
Add php-web-simple devfiles

### PR DESCRIPTION
**Pull request by vrubezhny**
Signed-off-by: Victor Rubezhny <vrubezhny@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
Seems that https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/codeready-workspaces/f0fd2b3f9e5fe581dfcb9f0f3a9b72fff8c88072/dependencies/che-devfile-registry/devfiles/php-web-simple/devfile.yaml doesn't work OOTB because of the registry.redhat.io authentication requirement

But if you add a registry using your auth secret (copied from ~/.docker/config.json if you have it):

image

Then ...

image

**Comment by nickboldt** [comment]
however seems like the apache setup isn't working:

image

image

**Comment by vrubezhny** [comment]
@nickboldt I've updated the PR, please review.

File: dependencies/che-devfile-registry/devfiles/php-web-simple/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+displayName: "PHP Simple"
+description: PHP Stack with PHP 7.1 and simple web application
+tags: ["PHP", "Apache", "Debian"]
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
```
Comment (tolusha) [inline dependencies/che-devfile-registry/devfiles/php-web-simple/meta.yaml]: Should be replaced by a proper image from
https://github.com/redhat-developer/codeready-workspaces/tree/master/ide/codeready-ide-stacks/src/main/resources/stacks-images

Comment (vrubezhny) [reply dependencies/che-devfile-registry/devfiles/php-web-simple/meta.yaml]: Done

**Pull request merged**